querydb/src/main/scala/io/joern/scanners/php/PhpJoern.scala

```diff
@@ -0,0 +1,144 @@
+  def codei()(implicit context: EngineContext): Query =
+    Query.make(
+      name = "phpjoern_codei",
+      author = PHPJOERN_EXT_AUT,
+      title = "CWE-94(Code Injection): A parameter is used in an insecure codei related func call.",
+      description = """
+          |An attacker controlled parameter is used in an insecure codei related func call.
+          |
+          |If the parameter is not validated and sanitized, this is a remote code execution.
+          |""".stripMargin,
+      score = 5,
+      withStrRep({ cpg =>
+        // $_REQUEST["foo"], $_GET["foo"], $_POST["foo"]
+        // are identifier (at the moment)
+        def source =
+          cpg.call.name(Operators.assignment).argument.code(".*_(REQUEST|GET|POST).*")
+
+        def sink = cpg.call.name("(eval|assert|create_function|include|include_once|require|require_once|call_user_func|call_user_func_array)").argument
+
+        sink.reachableBy(source).l.iterator
+      }),
+      tags = List(QueryTags.remoteCodeExecution, QueryTags.default, PHPJOERN_TAG)
+    ) 
+
+  @q
+  def uuf()(implicit context: EngineContext): Query =
+    Query.make(
+      name = "phpjoern_uuf",
+      author = PHPJOERN_EXT_AUT,
+      title = "CWE-434(Unrestricted Upload of File): A parameter is used in an insecure uuf related func call.",
+      description = """
+          |An attacker controlled parameter is used in an insecure uuf related func call.
+          |
+          |If the parameter is not validated and sanitized, this is a remote code execution.
+          |""".stripMargin,
+      score = 5,
+      withStrRep({ cpg =>
+        // $_REQUEST["foo"], $_GET["foo"], $_POST["foo"]
+        // are identifier (at the moment)
+        def source =
+          cpg.call.name(Operators.assignment).argument.code(".*_(REQUEST|GET|POST).*")
+
+        def sink = cpg.call.name("(file_get_contents|readfile|fgets|file|fopen|file_put_contents|fwrite|move_uploaded_file|unlink|rename|chmod|chown)").argument
+
+        sink.reachableBy(source).l.iterator
+      }),
+      tags = List(QueryTags.default, PHPJOERN_TAG)
+    ) 
+
+
+  @q
+  def xss()(implicit context: EngineContext): Query =
+    Query.make(
+      name = "phpjoern_xss",
+      author = PHPJOERN_EXT_AUT,
+      title = "CWE-79(Cross-site Scription): A parameter is used in an insecure xss related func call.",
+      description = """
+          |An attacker controlled parameter is used in an insecure xss related func call.
+          |
+          |If the parameter is not validated and sanitized, this is a XSS.
+          |""".stripMargin,
+      score = 5,
+      withStrRep({ cpg =>
+        // $_REQUEST["foo"], $_GET["foo"], $_POST["foo"]
+        // are identifier (at the moment)
+        def source =
+          cpg.call.name(Operators.assignment).argument.code(".*_(REQUEST|GET|POST).*")
+
+        def sink = cpg.call.name("(assert|echo|exit|print|printf|vprintf|print_r|var_dump)").argument
+
+        sink.reachableBy(source).l.iterator
```

Comment:
This should be an iterator by default I believe
```suggestion
        sink.reachableBy(source)
```